values/kube-prometheus-stack.yml

```
@@ -54,8 +54,8 @@ prometheusOperator:
 %{~ endif ~}
 
 prometheus:
-  prometheusSpec:
   %{~ if prometheus_resources.cpu_requests != "" && prometheus_resources.memory_requests != "" ~}
+  prometheusSpec:
```

If I understand correctly the `storageSpec` should be under the `prometheusSpec` node, so I think the old positions were correct:

```suggestion
  prometheusSpec:
  %{~ if prometheus_resources.cpu_requests != "" && prometheus_resources.memory_requests != "" ~}
```

---

Done